dialog/file.go

```diff
@@ -151,7 +156,8 @@ func (f *fileDialog) makeUI() fyne.CanvasObject {
 	scrollBread := widget.NewScrollContainer(f.breadcrumb)
 	body := fyne.NewContainerWithLayout(layout.NewBorderLayout(scrollBread, nil, nil, nil),
 		scrollBread, f.fileScroll)
-	header := widget.NewLabelWithStyle(label+" File", fyne.TextAlignLeading, fyne.TextStyle{Bold: true})
+	headerLabel := widget.NewLabelWithStyle(label+" File", fyne.TextAlignLeading, fyne.TextStyle{Bold: true})
```

Comment:
I don't think that adding a non-essential configuration belongs in the header

Reply:
I agree. Suggestions for an alternative location?

Reply:
moved to footer

Reply:
Isn't that where the current file name is shown? if so it takes quite a bit of space away from that.

Reply:
Doesn't appear to be, unless I'm missing something

<img width="756" alt="Screen Shot 2020-09-04 at 10 37 36 PM" src="https://user-images.githubusercontent.com/10604101/92296130-4d597180-eeff-11ea-9320-7ec8a2f96579.png">

Reply:
I think you've removed it, should look like this: 
<img width="867" alt="Screenshot 2020-09-07 at 22 58 13" src="https://user-images.githubusercontent.com/294436/92418094-a7f00900-f15d-11ea-8f9e-ec7e677ecdad.png">

Reply:
Yep. I didn't provide the check in the footer layout. Move to top:

<img width="756" alt="Screen Shot 2020-09-08 at 7 39 06 PM" src="https://user-images.githubusercontent.com/10604101/92537873-366e9400-f20b-11ea-9ef1-68a7f0767f1e.png">